crates/stackable-certs/src/ca/mod.rs

```diff
@@ -468,19 +498,57 @@ fn format_leaf_certificate_subject(name: &str, scope: &str) -> Result<Name> {
 
 #[cfg(test)]
 mod tests {
+    use const_oid::ObjectIdentifier;
+
     use super::*;
 
+    const TEST_CERT_LIFETIME: Duration = Duration::from_hours_unchecked(1);
+    const TEST_SAN: &str = "airflow-0.airflow.default.svc.cluster.local";
+
     #[tokio::test]
     async fn rsa_key_generation() {
         let mut ca = CertificateAuthority::new_rsa().unwrap();
-        ca.generate_rsa_leaf_certificate("Airflow", "pod", Duration::from_secs(3600))
-            .unwrap();
+        let cert = ca
+            .generate_rsa_leaf_certificate("Airflow", "pod", [TEST_SAN], TEST_CERT_LIFETIME)
+            .expect("RSA certificate generation failed");
```

Comment:
praise: Nicely done!

note: The message should instead read _why_ this will never fail.

Reply:
I don't know why https://github.com/stackabletech/operator-rs/blob/34588a2bed6a85701db6a25fab65648c8c65878c/crates/stackable-certs/src/keys/ecdsa.rs#L33 is fallible.

I'm basically treating this as error handling similar to snafu, where panicing is allowed. I have no clue why this could fail and I'm not comfortable making a claim why this would never fail for all future version that might come

Reply:
> I have no clue why this could fail and I'm not comfortable making a claim why this would never fail for all future version that might come

Then we should make it seem like we know by using `expect`. Using `unwrap` would be a better fit then.